services/ows_refactored/surface_temperature/ows_lsc2_st_cfg.py

```diff
@@ -0,0 +1,149 @@
+from ows_refactored.common.ows_reslim_cfg import reslim_landsat
+
+bands_ls5_st = {"ST_B6": ["st", "surface_temperature"]}
+bands_ls7_st = {"ST_B6": ["st", "surface_temperature"]}
+bands_ls8_st = {"ST_B10": ["st", "surface_temperature"]}
+
+style_lsc2_st = {
+    "name": "surface_temperature",
+    "title": "Surface temperature - Celsius",
+    "abstract": "Surface temperature in degrees Celsius",
+    "index_expression": "(0.00341802*st - 124.15)",
+    "mpl_ramp": "magma",
```

Comment:
I think this is your problem.  If you want to use `mpl_ramp`, you must also set the `range`.